pod/perldiag.pod

```diff
@@ -5467,6 +5467,33 @@ character class, just escape the square brackets with the backslash: "\[="
 and "=\]".  The S<<-- HERE> shows whereabouts in the regular expression the
 problem was discovered.  See L<perlre>.
 
+=item Possible attempt to escape whitespace in qw() list
+
+(W qw) qw() lists contain items separated by whitespace; contrary to
+what some might expect, backslash characters cannot be used to "protect"
+whitespace from being split, but are instead treated at literal data.
```

Comment:
Don't you want `treated as` rather than `treated at`?

Reply:
Yes, this is a typo. Thanks.